Dockerfile

```diff
@@ -14,6 +14,12 @@ RUN curl -sL https://deb.nodesource.com/setup_16.x | bash - && \
     pip install --no-cache-dir -r requirements/dev-requirements.txt && \
     # install mlflow in editable form
     pip install --no-cache-dir -e . && \
+    groupadd -g 10001 mlflow && \
```

Comment:
I think to make this most compatible with all potential systems where users would execute this, this command should be run separately prior to the environment declaration. 
A declaration of the explicit shell to use would be good as well instead of just setting the home path for the user. 

For a solid reference that's used pretty widely, let's adopt the practices of node: https://github.com/nodejs/docker-node/blob/6bae0eb339bc5be880f01b652f48fdf939f76590/12/buster/Dockerfile#L3-L4

Reply:
Sounds good, I'll update..